src/formBuilder/utils.js

```diff
@@ -1495,6 +1508,15 @@ export function generateElementComponentsFromSchemas(parameters: {
   return elementList;
 }
 
+export function cleanUiSchema(newName, oldName, uiSchema) {
+  if (
+    (newName !== oldName && !oldName.startsWith('ui:')) ||
+    newName == '__delete'
```

Comment:
Not sure if the use of double equals instead of triple is intentional here? The double will only compare if the value is the same whereas triple will also compare if the type is the same too.